_data/mentors.yml

```
@@ -1303,7 +1333,7 @@
   index: 43
```

```suggestion
  index: 44
```

There are two mentors with index 43